roles/nginxplus/templates/default_nginx.conf.j2

```diff
@@ -0,0 +1,69 @@
+{{ ansible_managed | comment }}
```

Comment:
I might rename this template (which would also require changing the name in the task that uses it). I don't think it's really a default config any more.

Reply:
🤔 I am pretty sure it is used. (certainly on the dev environment)